homeassistant/components/denonavr/media_player.py

```diff
@@ -249,6 +249,19 @@ def __init__(
             and MediaPlayerEntityFeature.SELECT_SOUND_MODE
         )
 
+        self._receiver.register_callback("ALL", self._telnet_callback)
+
+    async def _telnet_callback(self, zone, event, parameter):
+        """Process a telnet command callback."""
+        if zone != self._receiver.zone:
+            return
+
+        self.async_schedule_update_ha_state(False)
+
+    async def async_will_remove_from_hass(self) -> None:
+        """Clean up the entity."""
+        self._receiver.unregister_callback("ALL", self._telnet_callback)
+
     @async_log_errors
     async def async_update(self) -> None:
         """Get the latest status information from device."""
```

Comment:
When the telnet connection is up and running, we don't need to update the entire receiver anymore, but only attributes like interpret, title etc. which are not covered by telnet. This would save some API calls.

```suggestion
        """Get the latest status information from device."""
        if self._receiver.telnet_connected is True and self._receiver.telnet_healthy is True
            await self._receiver.input.async_update_media_state()
            return
```

Reply:
Resolved in https://github.com/home-assistant/core/pull/85980/commits/b5f983ec35264389e2eb7e3417e31d6a208ec700